### PR TITLE
Added demo docs and sync workflow

### PR DESCRIPTION
**Pull request by zoey-kaiser**
WIP

### CODE REVIEW

**Comment by zoey-kaiser** [comment]
This PR currently runs a Github Action called "Sync Docs". What it does it open a PR on https://github.com/sidebase/docs.

It adds:

Files from `.docs/content` to `content/nuxt-session` - For the READMEs
Files from `.docs/public` to `public` - For any Images used in the docs

Once you merge the new docs in the repestive module repo, you can merge the created PR, to pull the changes onto the Website. We can then add a Action there, to update the website automatically!

**Comment by BracketJohn** [comment]
Thanks!

- Is there a remaining reason for putting the docs into a "secret" `.docs` folder?
- How can we enable "auto-merge" so that the syncing does not need manual action?
- Are you blocked here by me in any way atm?

**Comment by zoey-kaiser** [comment]
> Is there a remaining reason for putting the docs into a "secret" `.docs` folder?

I was doing it the way that the docus repo did it: https://github.com/nuxt-themes/docus/tree/dev/.docs
But in the end we can also make it just `docs`!

> How can we enable "auto-merge" so that the syncing does not need manual action?

I was thinking about this. I assume we could enable `auto-merge` from GitHub. Meaning, if there are no conflicts and the CI passes, it would get merged. I can play with these settings and find a way to get it working!

> Are you blocked here by me in any way atm?

I would not say so. I just wanted to ensure you had a look at my PoC. If you are happy with the implementation we can move forward to:

- Getting PRs like this set up in every repo. 
- Writing the docs for every repo. 
- Deploying a beta on v1.sidebase.io

An issue I am still facing in this repo is this: https://github.com/nuxt-themes/docus/issues/680. The docus team recommended we upgrade the module to Nuxt3 stable. I will add a new PR in nuxt-auth and see if that resolves it, as we already upgraded there!

**Comment by BracketJohn** [comment]
>  But in the end we can also make it just docs!

yeah,. let's move to just docs

> I was thinking about this. I assume we could enable auto-merge from GitHub. Meaning, if there are no conflicts and the CI passes, it would get merged. I can play with these settings and find a way to get it working!

this is essential, else the whole sync-workflow will not have solved the problem that contributions to docs in repo A need manual steps (that will be forgotten or too tedious for outside / inside collaborators) in order to go-live.

> I would not say so. I just wanted to ensure you had a look at my PoC. If you are happy with the implementation we can move forward to:

let's adapt thjis a bit:
1. get auto-sync without human inference to work (remember: this was a requirement I posed to accept this solution)
2. get this pr merged
3. see if the sync workflow works
4. test out `npm run build && npm run preview` in the `docs` repo to see that everything works in theory
5. then in parallel:
    - get @DeusAvalon to deploy it into the cluster with CD
    - do the docs PRs on other repos

File: docs/public/nuxt-session.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a test asset to move to the docs folder.
```
Comment (BracketJohn) [inline docs/public/nuxt-session.txt:1]: should this go?

File: docs/.env.example [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Create one with no scope selected on https://github.com/settings/tokens/new
+# This token is used for fetching the repository releases.
+GITHUB_TOKEN=
```
Comment (BracketJohn) [inline docs/.env.example:3]: missing newline here?

**Review by BracketJohn — COMMENTED** [review]
This PR is a lot bigger than https://github.com/sidebase/nuxt-parse/pull/3 - is this on purpose?

**Comment by zoey-kaiser** [comment]
> This PR is a lot bigger than [sidebase/nuxt-parse#3](https://github.com/sidebase/nuxt-parse/pull/3) - is this on purpose?

In the `nuxt-parse` PR the `package.lock.json` was missing. Therefore the big difference. As docus is currently not working anyway in these PRs and will need to be refactored, we could either delete the package-lock or ignore it.

File: docs/.env.example (hunk 2 above)
Comment (zoey-kaiser) [reply docs/.env.example:3]: I would delete this file anyway, as it was only included in the bootstrapped docus project.

File: docs/public/nuxt-session.txt (hunk 1 above)
Comment (zoey-kaiser) [reply docs/public/nuxt-session.txt:1]: I wanted to keep these in here to already have the public folder. I was scared that without it people would not know where to put their images for the docs.

**Comment by BracketJohn** [comment]
Let's ignore and keep it for now, thanks. Please delete the env file so that we can contuinue

**Comment by zoey-kaiser** [comment]
> Let's ignore and keep it for now, thanks. Please delete the env file so that we can contuinue

Deleted!

**Review by BracketJohn — APPROVED** [review]

**Pull request merged**